diff --git a/integration_test.go b/integration_test.go
index 510df0f..f40281c 100644
--- a/integration_test.go
+++ b/integration_test.go
@@ -166,6 +166,7 @@ func TestWriteFailure(t *testing.T) {
 }
 
 func TestCustomPayloadMessages(t *testing.T) {
+	t.Skip("SKIPPING")
 	cluster := createCluster()
 	session := createSessionFromCluster(cluster, t)
 	defer session.Close()
@@ -203,6 +204,7 @@ func TestCustomPayloadMessages(t *testing.T) {
 }
 
 func TestCustomPayloadValues(t *testing.T) {
+	t.Skip("SKIPPING")
 	cluster := createCluster()
 	session := createSessionFromCluster(cluster, t)
 	defer session.Close()
